docs/tbdev_getting_started.ipynb

```diff
@@ -1,235 +1,272 @@
 {
```

Comment:
<p>This doesn't currently work in Colab as I don't think TF/TB 2.3 is the native. I am not sure if this happens automatically or whether we need to do something. A workaround is to upgrade the version of TensorBoard at the start of the notebook, but it would be better if we can avoid it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/3920/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>